indico/util/user.py

```diff
@@ -96,6 +97,12 @@ def principal_from_fossil(fossil, allow_pending=False, allow_groups=True, allow_
         if group.group is None and not allow_missing_groups:
             raise ValueError('Multipass group does not exist: {}:{}'.format(provider, id_))
         return group
+    elif type_ == 'EventRole':
+        role = EventRole.get(id_)
```

Comment:
Do we ensure the role belongs to the same event somewhere? Maybe we should add an optional `event` arg to this function and only accept event roles if it's not None (like with the `allow_*` args) and the role belongs to the same event...